### PR TITLE
Pull request testing

### PR DESCRIPTION
**Pull request by linh54**
Pull request testing
_see italics_

__see resolves__

### CODE REVIEW

**Review by ktandabany — APPROVED** [review]
Approved

File: _slides/07-linh54.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
----
-layout: slide
-title: "Welcome to our slide deck!"
----
-
-Caption here
-
```
Comment (rjiang13) [inline _slides/07-linh54.md:7]: Approved. This is great.

**Review by mandadi277 — APPROVED** [review]
Approved

**Review by rjiang13 — APPROVED** [review]
Looks good.

**Pull request merged**